controllers/apply/awards-for-all/fields.js

```diff
@@ -300,7 +293,324 @@ module.exports = function fieldsFor({ locale, data = {} }) {
         return { ...defaultProps, ...props };
     }
 
-    function organisationTypeField() {
+    function fieldProjectDateRange() {
```

Comment:
I reckon it would make sense to extract all the fields into `field*` functions like we do with the `step*` functions in `form.js` but I'd much rather leave that until after we've added the welsh translation and we can see how noisy this file gets.